spacy_udpipe/udpipe.py

```diff
@@ -0,0 +1,202 @@
+import re
+from typing import Dict, List, Optional, Union
+
+from ufal.udpipe import (InputFormat, Model, OutputFormat, ProcessingError,
+                         Sentence)
+
+from .utils import get_path
+
+
+class PretokenizedInputFormat(object):
+    """Dummy tokenizer for pretokenized input.
+
+    Execution speed might be slow compared to other UDPipe tokenizers
+    due to pure Python implementation. Mocks InputFormat API to enable
+    plug-and-play behaviour.
+    """
+    NO_SPACE = "SpaceAfter=No"
+
+    def setText(self, text: str) -> None:
+        """Store text in iterable lines for tokenization.
+
+        text: string, where each sentence is on a line and tokens
+              are separated by tabs.
+        """
+        self.lines = iter(text.split("\n"))
+
+    def nextSentence(self, sentence: Sentence, _: ProcessingError) -> bool:
```

Comment:
Why do you have this ProcessingError parameter?

Reply:
Custom `PretokenizedInputFormat` complies with the [UDPipe Tokenizer API](http://ufal.mff.cuni.cz/udpipe/api-reference#input_format_next_sentence) to enable easy plug-and-play with the rest of the code.